src/main/kotlin/fm/unit/UnitApp.kt

```diff
@@ -61,6 +64,19 @@ fun Application.module() {
         return content.toString()
     }
 
+    // Database setup
+    val db_user =  "kjeschkies" //System.getenv("POSTGRES_USER")
```

Comment:
It's ok for now, but should we introduce [application.conf](https://ktor.io/servers/configuration.html) with those values? Can we then remove these from `build.gradle.kts` too?